build.sh

```diff
@@ -54,7 +54,7 @@ sed -i -e "s/BUILD_DATE/${DATE}/g" maestroutils/status.go
 # color()(set -o pipefail;"$@" 2>&1>&3|sed $'s,.*,\e[31m&\e[m,'>&2)3>&1
 
 #Developers may have a GOBIN defined. If not define it to ./bin
-if [ -z "${GOBIN}" ]; then
+if [[ -z "${GOBIN:-}" ]]; then
```

Comment:
```suggestion
if [ -z "${GOBIN:-}" ]; then
```

`-z` is a builtin check. The extra `[]` is mainly used for direct string comparisons.

Reply:
I was actually trying to be consistent with the file :https://github.com/armPelionEdge/maestro/pull/32/files#diff-0b83f9dedf40d7356e5ca147a077acb4L64

Can remove, no worries 👍